build-linux.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash 
+# simple build script for linux
+
+#package list for debian 9.x+
+deb="git build-essential libsdl2-2.0-0 libsdl2-dev libjansson-dev libexpat1-dev libcurl4-openssl-dev libpng-dev libjpeg-dev libspeex-dev libspeexdsp-dev"
+#package list for debian 8.x
+deb_old="git build-essential libsdl2-2.0-0 libsdl2-dev libjansson-dev libexpat1-dev libcurl4-openssl-dev libpng12-dev libjpeg-dev libspeex-dev libspeexdsp-dev"
+#package list for fedora/el
+rpm="vim-common pcre-devel mesa-libGL-devel SDL2-devel make gcc jansson-devel expat-devel libcurl-devel libpng-devel libjpeg-turbo-devel speex-devel speexdsp-devel"
+
+buildlog="/tmp/ezquake-build.log"
+
+if [ -e "$buildlog" ];then
+	rm -f "$buildlog"
+fi
+
+debian=0
+
+#ansi color codes
+RED='\e[31m'
+GREEN='\e[32m'
+NC='\e[0m'
+
+trap "if [ \$? -ne 0 ];then echo -e \"${RED}Failure, bailing out.${NC}\";if [ -e $buildlog ];then echo \"Please see $buildlog for details.\";fi;exit 0;fi" INT TERM EXIT
+
+if [ `id -u` -ne 0 ];then
+	echo "You must be root to run this script as it interfaces with your package manager, run with sudo or root privileges."
+	exit 1
+fi
+
+if [ ! -e Makefile ];then
+	echo "Cannot find Makefile, please run this script from the source-code directory."
+	exit 1
+fi
+
+function cpu_cores() {
```

Comment:
You can call `nproc` to retrieve the number of CPU threads. Consider building with `(($(nproc) * 3/2))` threads (150% of CPU threads, which is a bit faster). The operation looks a bit funny (and it only works in Bash), but that's because Bash does not feature floating-point arithmetic :smile: